### DISCUSSION TITLE
calc 本地评测数据更新

### DISCUSSION THREAD

**Discussion by CircuitCoder** [physics-data/faq#105] [General] [answered]
由于生成数据时的疏忽，Task2 的本地测试数据有误（和 Task2 题面不符，但是可以认为是 Task3 的正确数据）

助教生成了新的测试数据，请使用 git apply 应用这个 [patch](https://gist.github.com/CircuitCoder/782d004812ade55acd8a9b4ad117f2e1)（`git apply 0001-some.patch` 或者 `git apply` 然后将 Patch 内容粘贴进标准输入）

额外的，你也可以直接从 https://github.com/physics-data/tpl_calc 更新 /data/task2 目录。当然，你可以尝试自己造一些数据进行测试，或者直接完成 Task3——Task2是它的一个子集。

**Comment by heroxbd** [comment, marked as answer]
感谢喵喵。
